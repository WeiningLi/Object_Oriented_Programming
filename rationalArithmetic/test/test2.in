create a -5/2
print a
create b 5/2
print b
+ a b
+ a b
+ b b
- b a
* a b
/ b a
+= a b
-= a b
u- a
- b b
- a b
+ a a
* a a
/ a a
+= a a
-= a a
