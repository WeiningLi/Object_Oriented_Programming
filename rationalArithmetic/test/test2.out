-5/2
5/2
0
0
5
5
-25/4
-1
0
-5/2
5/2
0
-5
-5
25/4
1
-5
0
